Cpp/Library/Processor/ProcessorToolbox.cc

```diff
@@ -331,7 +301,7 @@ namespace Nymph
         if( it == fProcMap.end() )
         {
             LWARN( proclog, "Processor <" << procName << "> was not found." );
-            return NULL;
+            return nullptr;
```

Comment:
I think this should return `false`, but it does not matter really.

Reply:
I'm pretty sure this should be as it is, returning a `nullptr`.  This is in `ReleaseProcessor()`, which in addition to removing the processor from the toolbox also returns it to the client.  In `RemoveProcessor()` we instead return a bool because we're just getting rid of the processor entirely.

Reply:
Oh, I was fooled by the git view that compares changes between the two versions, which made it look like it is in `RemoveProcessor`. My bad.

Reply:
Oh, I was fooled by the git view that compares changes between the two versions, which made it look like it is in `RemoveProcessor`. My bad.